### DISCUSSION TITLE
Обсуждение дополнения simpleopencart for unishop2

### DISCUSSION THREAD

**Discussion by get-web** [get-web/simpleopencart_for_unishop2_by_get-web#1] [Announcements]
## 👋 Welcome!
Тема для обсуждения дополнения simpleopencart for unishop2

**Comment by nogocuHoBuk** [comment]
Привет.
Фикс вывода описания платежных систем:
catalog/view/theme/unishop2/template/checkout/simplecheckout_payment.twig
строки 48 и 50
Вместо **quote.description** нужно **payment_method.description**
Результат:
![Screenshot_2](https://github.com/get-web/simpleopencart_for_unishop2_by_get-web/assets/55383967/252c44d3-e4a2-4261-8cef-6017bec57f23)

**Reply by get-web** [reply]
Спасибо. Обновил.

**Reply by nogocuHoBuk** [reply]
Ну и небольшое "пожелание".
Свои стили лучше "засунуть" в отдельный файл и подключать их в начале кода (вместе с унишоповским стилем), А то в текущей ситуации возникает момент, когда форма загрузилась, но стили ещё не применились. Раздражающий слегка фактор. :)

**Reply by nogocuHoBuk** [reply]
И ещё один момент. 
В модификатор добавить языковые переменные унишопа (перед загрузкой симпловых)
![image](https://github.com/get-web/simpleopencart_for_unishop2_by_get-web/assets/55383967/4cb4c255-6ae5-4f75-90a3-3bd93efa4033)
  Ну и я добавил блок замены текста корзины в шапке (ибо там тоже вываливалось "товаров: 5 (сумма)" вместо лаконичного количества товаров.
  
Только в коде нужно будет заменить в 
catalog/view/theme/unishop2/template/checkout/simplecheckout.twig и catalog/view/theme/unishop2/template/checkout/simplecheckout_cart.twig
unicheckout_title на text_product_total
unicheckout_heading на  text_your_order
И тогда не будет необходимости менять языковые файлы модификатором. Всё подтянется (Ибо есть же не только RU и EN)
Мой итоговый вариант модификатора как пример (а то не получается корректно код сюда вставить):
[xml.zip](https://github.com/get-web/simpleopencart_for_unishop2_by_get-web/files/11524581/xml.zip)

Возможно я тоже что-то упустил, но мне этого хватило :)

**Reply by get-web** [reply]
> И ещё один момент. В модификатор добавить языковые переменные унишопа (перед загрузкой симпловых)> 

Это хорошее решение

**Comment by nogocuHoBuk** [comment]
Ещё одна небольшая правка файла catalog/view/theme/unishop2/template/checkout/simplecheckout_cart.twig
На основе того, что выложен в телегу: https://t.me/unishop2_chat/56858
Там же в треде расписал, дублирую тут:
вместо

     {% for total in totals %}
            <div class="unicheckout-sticky__total-item total">
                <div class="unicheckout-sticky__total-title">
                    {{ total.title }}
                </div>
                <div class="unicheckout-sticky__total-text">
                    <span class="{{ total.code == 'total' ? 'total' : 'sub_total' }}">{{ total.text }}</span>
                </div>
            </div>
    {% endfor %} 
Предлагаю вот так:

     {% for total in totals %}
            <div class="unicheckout-sticky__total-item {{ total.code }}">
                <div class="unicheckout-sticky__total-title">
                    {{ total.title }}
                </div>
                <div class="unicheckout-sticky__total-text">
                    <span class="{{ total.code }}">{{ total.text }}</span>
                </div>
            </div>
    {% endfor %}

Результат - красные скидки и жирным только тотал:
![image](https://github.com/get-web/simpleopencart_for_unishop2_by_get-web/assets/55383967/dc3eb925-7222-4612-b864-448833e315c0)

**Comment by oldpapa69** [comment]
В Симпл отключен переключатель отображать колонку модели, но она все равно отображается.

**Reply by get-web** [reply]
Исправил
